modular_skyrat/modules/assault_operatives/code/assault_operatives.dm

```diff
@@ -0,0 +1,260 @@
+		if("medic")
+			chosen_loadout_type = /datum/outfit/assaultops/medic
+			loadout_desc = "<span class='notice'>You have chosen the Medic class, your role is providing medical aid to fellow operatives!</span>"
+		if("heavy")
+			chosen_loadout_type = /datum/outfit/assaultops/heavy
+			loadout_desc = "<span class='notice'>You have chosen the Heavy class, your role is continuous suppression!</span>"
+		if("assault")
+			chosen_loadout_type = /datum/outfit/assaultops/assault
+			loadout_desc = "<span class='notice'>You have chosen the Assault class, your role is general combat!</span>"
+		if("sniper")
+			chosen_loadout_type = /datum/outfit/assaultops/sniper
+			loadout_desc = "<span class='notice'>You have chosen the Sniper class, your role is suppressive fire!</span>"
+		if("tech")
+			chosen_loadout_type = /datum/outfit/assaultops/tech
+			loadout_desc = "<span class='notice'>You have chosen the Tech class, your role is hacking!</span>"
+		else
+			chosen_loadout_type = pick(/datum/outfit/assaultops/cqb, /datum/outfit/assaultops/demoman, /datum/outfit/assaultops/medic, /datum/outfit/assaultops/heavy, /datum/outfit/assaultops/assault, /datum/outfit/assaultops/sniper, /datum/outfit/assaultops/tech)
+
+	if(!chosen_loadout)
+		chosen_loadout_type = /datum/outfit/assaultops
+
+	for(var/obj/item/item in human_target.get_equipped_items(TRUE))
+		qdel(item)
+
+	var/obj/item/organ/brain/human_brain = human_target.getorganslot(BRAIN)
+	human_brain.destroy_all_skillchips() // get rid of skillchips to prevent runtimes
+
+	human_target.equipOutfit(chosen_loadout_type)
+
+	human_target.regenerate_icons()
+
+	to_chat(human_target, loadout_desc)
+
+	equipped_class = chosen_loadout
+	return TRUE
+
+/datum/antagonist/assault_operative/proc/equip_operative()
+	if(!ishuman(owner.current))
+		return
+	var/mob/living/carbon/human/human_target = owner.current
+	for(var/obj/item/item in human_target.get_equipped_items(TRUE))
+		qdel(item)
+
+	var/obj/item/organ/brain/human_brain = human_target.getorganslot(BRAIN)
+	human_brain.destroy_all_skillchips() // get rid of skillchips to prevent runtimes
+
+	human_target.equipOutfit(assault_operative_default_outfit)
+
+	human_target.regenerate_icons()
+
+	return TRUE
+
+/datum/antagonist/assault_operative/create_team(datum/team/assault_operatives/new_team)
+	if(!new_team)
+		if(!always_new_team)
+			for(var/datum/antagonist/assault_operative/assault_operative in GLOB.antagonists)
+				if(!assault_operative.owner)
+					stack_trace("Antagonist datum without owner in GLOB.antagonists: [assault_operative]")
+					continue
+				if(assault_operative.assault_team)
+					assault_team = assault_operative.assault_team
+					return
+		assault_team = new /datum/team/assault_operatives
+		assault_team.add_member(owner)
+		assault_team.update_objectives()
+		return
+	if(!istype(new_team))
+		stack_trace("Wrong team type passed to [type] initialization.")
```

Comment:
Well, in this case, you'd probably want to return, no? Unless stack_trace() already returns, but I got a feeling it does.

Reply:
Stack trace returns afaik